### ISSUE TITLE
ASDF complains about the leading zeros in the version number

### ISSUE THREAD

**Issue by nklein** [wmannis/cl-svg#17]
The current version number in the `cl-svg.asd` file is `"0.03"`. ASDF translates that into `(list 0 3)` and then translates that back to `"0.3"`. ASDF then emits warnings indicating the version number has leading zeros.

```
; Warning: PARSE-VERSION: "0.03" contains leading zeros
; Warning: Invalid :version specifier "0.03" for component "cl-svg" from file #P"/Users/pat/quicklisp/dists/quicklisp/software/cl-svg-20180228-git/cl-svg.asd", using NIL instead
```

**Comment by nklein** [comment]
Created pull-request #18  for this.

**Comment by nklein** [comment]
Corrected in pull-request #19
